articles/aks/ai-and-ml-overview.md

```diff
@@ -0,0 +1,13 @@
+## AI/ML Workloads Overview 
```

Comment:
Wondering if somewhere throughout this we want to include anything similar to what Amazon has done [here](https://docs.aws.amazon.com/eks/latest/userguide/machine-learning-on-eks.html)? Couldn't find an equivalent for AI but wanted to share this anyway

Reply:
I like that suggestion t but I think it would repetitive to some of the content we have in the body of the page. 

Lets me think about if there's a place for both - which may require a reorganization of the content.